src/components/KillButton.js

```diff
@@ -6,6 +6,9 @@ import { query, updateDoc, collection, getDocs, where } from "firebase/firestore
 const KillButton = (props) => {
     const [selectedPlayer, setSelectedPlayer] = useState('');
     const [showAlert, setShowAlert] = useState(false); // State to control the alert visibility
+    const [editableArray, setEditableArray] = useState([]);
```

Comment:
Q: Does this editableArray need to be a state variable, and if so, where is it getting used? I might just be missing something here

Reply:
I removed them in the most recent commit. I may have used them initially in an earlier commit I think, but not in use anymore.